xbmc/dialogs/GUIDialogSimpleMenu.cpp

```diff
@@ -90,12 +91,31 @@ bool CGUIDialogSimpleMenu::ShowPlaySelection(CFileItem& item, const std::string&
 
   CFileItemList items;
 
-  if (!GetDirectoryItems(directory, items, XFILE::CDirectory::CHints()))
+  if (item.GetVideoContentType() == VideoDbContentType::EPISODES)
   {
-    CLog::Log(LOGERROR,
-              "CGUIWindowVideoBase::ShowPlaySelection - Failed to get play directory for {}",
-              directory);
-    return true;
+    // See if can simplify titles
+    // Get episodes on disc
+    std::vector<CVideoInfoTag> episodes;
+    CVideoDatabase database;
+    database.Open();
```

Comment:
Not sure why exactly, but we always check the return value of db.Open(), log an error in case it fails and return false.